app/uk/gov/hmrc/cgtpropertydisposals/models/des/SubscriptionStatus.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposals.models.des
+
+import play.api.libs.json.{JsError, JsString, JsSuccess, Reads}
+
+sealed trait SubscriptionStatus extends Product with Serializable
+
+object SubscriptionStatus {
+
+  final case object NotSubscribed extends SubscriptionStatus
```

Comment:
don't need `final` here - members of objects are all final except case classes.

Reply:
I've got rid of it now